mist-common/src/main/java/edu/snu/mist/common/operators/UnionOperator.java

```diff
@@ -217,4 +218,14 @@ public void processRightWatermark(final MistWatermarkEvent event) {
       drainUntilMinimumWatermark();
     }
   }
+
+  @Override
+  public void processLeftCheckpoint(final MistCheckpointEvent input) {
+    outputEmitter.emitCheckpoint(input);
```

Comment:
How to checkpoint the states of union operator?

Reply:
I will chagne UnionOperator to checkpoint its state.

Reply:
Actually, I don't think we need to checkpoint the UnionOperator.
We currently checkpoint the states of all stateful operators based on the minimum timestamp among most-recent-event-timestamp.
This way, there is no need to save the left/right upstream Queues, left/right MistWatermarkEvents, left/right recentLeftTimestamps of the UnionOperator.